src/batch-client/index.ts

```diff
@@ -460,6 +461,12 @@ export class ZimbraBatchClient {
 			}
 		});
 
+	public modifySearchQuery = (options: SearchFolderInput) =>
```

Comment:
Can we rename this to `modifySearchFolder` to match the name of the endpoint being used?

Reply:
agreed